### PR TITLE
fix: Avatar Group popover control focus & more button color

### PR DESCRIPTION
**Pull request by olshansky**
BREAKING CHANGE: 
- added `.fd-avatar-group__popover-control` class to `.fd-popover__control`
- `role` changed to `img`, added `alt` and `title` instead of `aria-label` for Avatars

## Related Issue
Closes: https://github.com/SAP/fundamental-styles/issues/2240

## Description
Fixed: 
- focus state of popover-control
- text and border color of "more button"

### Before:
image

### After:
image

#### Please check whether the PR fulfills the following requirements

1. The output matches the design specs
- [x] All values are in `rem`
- [x] Text elements follow the truncation rules
- [x] hover state of the element follow design spec
- [x] focus state of the element follow design spec
- [x] active state of the element follow design spec
- [x] selected state of the element follow design spec
- [x] selected hover state of the element follow design spec
- [x] pressed state of the element follow design spec
- [x] Responsiveness rules - the component has modifier classes for all breakpoints
- [x] Includes Compact/Cosy/Tablet design
- [x] RTL support
2. The code follows fundamental-styles code standards and style
- [x] only one top level `fd-*` class is used in the file
- [x] BEM naming convention is used
- [x] Mixins are used for repeatable code (`fd-rtl`, `fd-ellipsis`, `fd-flex`, `fd-selected`, `fd-focus`, ect.)
- [x] A11y support - keyboard support, screenreader support, proper ARIA attributes, etc.
- [n/a] `fd-reset()` mixin is applied to all elements
- [x] Variables are used, if some value is used more than twice.
- [x] Checked if current components can be reused, instead of having new code.
3. Testing
- [n/a] tested Storybook examples with "CSS Resources" `normalize` option 
- [n/a] tested Storybook examples with "CSS Resources" `unnormalize` option 
- [n/a] Verified all styles in IE11
- [x] Updated tests
- [x] last commit message should have `[ci visual]` so it can trigger chromatic visual regression (e.g. `test: run chromatic visual regression [ci visual]`)
4. Documentation
- [x] Storybook documentation has been created/updated
- [x] Breaking Changes [wiki](https://github.com/SAP/fundamental-styles/wiki/Breaking-Changes) has been updated in case of breaking changes.

### CODE REVIEW

**Review by platon-rov — APPROVED** [review]

**Comment by InnaAtanasova** [comment]
@olshansky looks great! You need to add in the template
```
BREAKING CHANGE: added .fd-avatar-group__popover-control class to .fd-popover__control
```

and update the Wiki page for the Breaking changes (link is the last checkbox in the PR template". All changes to the markup and adding/removing classes are considered breaking changes as the implementing libraries (fund-ngx, fund-react, etc) need to integrate these changes with the newest release of fund-styles.

**Comment by puru-hk** [comment]
No alternative text available for screen reader to read in case of Avatars with Image and Icons.

**Comment by olshansky** [comment]
> No alternative text available for screen reader to read in case of Avatars with Image and Icons.

@puru-hk changed `role` and added `alt` for avatars

**Comment by olshansky** [comment]
> and update the Wiki page for the Breaking changes (link is the last checkbox in the PR template". All changes to the markup and adding/removing classes are considered breaking changes as the implementing libraries (fund-ngx, fund-react, etc) need to integrate these changes with the newest release of fund-styles.

@InnaAtanasova thank you, done

**Review by InnaAtanasova — APPROVED** [review]

File: stories/avatar-group/avatar-group.stories.js [hunk 1]
```diff
@@ -52,7 +52,7 @@ export const individualType = () => `<div class="fd-avatar-group fd-avatar-group
     <div class="fd-popover">
         <div class="fd-popover__control">
             <div class="fd-avatar-group__item" tabindex="0" aria-haspopup="true" aria-expanded="false" onclick="onPopoverClick('popover_avatar_5z28edb');">
-                <span class="fd-avatar fd-avatar--s fd-avatar--circle fd-avatar--border" aria-label="Wendy Wallace">WW</span>
+                <span class="fd-avatar fd-avatar--s fd-avatar--circle fd-avatar--border" role="img" alt="Wendy Wallace" title="Wendy Wallace">WW</span>
```
Comment (droshev) [inline stories/avatar-group/avatar-group.stories.js:55]: I think this should be documented as a breaking change @InnaAtanasova

**Review by droshev — CHANGES_REQUESTED** [review]
we need to agree whether this is a breaking change and if yes - to document it.
If the upstream libraries(e.g. need to change => a breaking change)

File: stories/avatar-group/avatar-group.stories.js (hunk 1 above)
Comment (InnaAtanasova) [reply stories/avatar-group/avatar-group.stories.js:55]: @droshev if you think that it should be documented, then it should be

Comment (olshansky) [reply stories/avatar-group/avatar-group.stories.js:55]: @droshev it was documented on the [wiki page](https://github.com/SAP/fundamental-styles/wiki/Breaking-Changes#avatar-group--changed-structure-of-uses-avatar-component-2261)
I've added it to the PR description, thanks

**Review by droshev — APPROVED** [review]
@olshansky make sure when you merge to do it as a breaking change

**Review by puru-hk — APPROVED** [review]

**Comment by puru-hk** [comment]
Looks good

**Pull request merged**